File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Departamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Departamento
    {
        [Key]
        public int idDepartamento { get; set; }

        [MaxLength(20)]
        public string nombreDepartamento { get; set; }

        public int idPais { get; set; }
        [ForeignKey("idPais")]
        public Pais Pais { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Departamento.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Departamento
    <Key>
    Public Property idDepartamento As Integer
    <MaxLength(20)>
    Public Property nombreDepartamento As String
    Public Property idPais As Integer
    <ForeignKey("idPais")>
    Public Property Pais As Pais
End Class
